### PR TITLE
Task/template update

### PR DESCRIPTION
**Pull request by sbilge**
- update template files
- update template to CLI

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -24,21 +21,134 @@ classifiers = [
...
+text = "Apache 2.0"
 
-Repository = "https://github.com/ghga-de/ghga_validator"
+[project.urls]
+Repository = "https://github.com/ghga-de/ghga-validator"
 
 [project.scripts]
-
-ghga_validator = "ghga_validator.__main__:run"
+my-microservice = "ghga_validator.__main__:run"
```
Comment (lkuchenb) [inline pyproject.toml]:
```suggestion
ghga-validator = "ghga_validator.__main__:run"
```

File: pyproject.toml [hunk 2]
```diff
@@ -24,21 +21,134 @@ classifiers = [
     "Topic :: Software Development :: Libraries",
     "Intended Audience :: Developers",
 ]
+name = "ghga_validator"
+version = "1.1.1"
+description = "GHGA Validator - a short description"
 dependencies = [
-    "typer ~= 0.7.0",
+    "typer >= 0.9.0",
+    "typer >= 0.9.0",
```
Comment (lkuchenb) [inline pyproject.toml]:
```suggestion
    "typer >= 0.9.0",
```

File: pyproject.toml [hunk 3]
```diff
@@ -24,21 +21,134 @@ classifiers = [
     "Topic :: Software Development :: Libraries",
     "Intended Audience :: Developers",
 ]
+name = "ghga_validator"
+version = "1.1.1"
+description = "GHGA Validator - a short description"
```
Comment (lkuchenb) [inline pyproject.toml]: Please restore original description

**Review by lkuchenb — CHANGES_REQUESTED** [review]

**Review by lkuchenb — CHANGES_REQUESTED** [review]
The REAMDE still has the template snippets, the previous information is gone. Feel free to kill the README generator, it really doesn't do much good for a CLI tool.

**Comment by sbilge** [comment]
> The REAMDE still has the template snippets, the previous information is gone. Feel free to kill the README generator, it really doesn't do much good for a CLI tool.

I removed the github check and revert the readme to its previous state. But I left the files that come with the template to autogenerate the readme, just for the sake of not diverging from the original template too much. Do you think I should also remove them?

**Review by lkuchenb — APPROVED** [review]

**Pull request merged**